x-pack/plugins/ml/server/models/file_data_visualizer/file_data_visualizer.js

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+import Boom from 'boom';
+import fs from 'fs';
+import os from 'os';
+const util = require('util');
+// const readFile = util.promisify(fs.readFile);
+const readdir = util.promisify(fs.readdir);
+const writeFile = util.promisify(fs.writeFile);
+
+export function fileDataVisualizerProvider(callWithRequest) {
+  async function analyseFile(data) {
```

Comment:
Should be `analyzeFile` for consistency with `DataVisualizer` I think, and the route which is `analyzeFile`